modules/core/src/v2/coins/sol.ts

```diff
@@ -50,6 +51,12 @@ export interface TransactionPrebuild extends BaseTransactionPrebuild {
   source: string;
 }
 
+export interface SolVerifyTransactionOptions extends VerifyTransactionOptions {
+  memo?: Memo;
+  feePayer?: string;
+  blockhash?: string;
```

Comment:
should these be required? every valid transaction needs a fee payer and blockhash